stage1_fly/process-credentials.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stage1_fly
+
+import (
+	"syscall"
+
+	"github.com/appc/spec/schema"
+	rktlog "github.com/rkt/rkt/pkg/log"
+	stage1commontypes "github.com/rkt/rkt/stage1/common/types"
+	stage1initcommon "github.com/rkt/rkt/stage1/init/common"
+)
+
+type ProcessCredentials struct {
+	uid               int
+	gid               int
+	supplementaryGIDs []int
+}
+
+func LookupProcessCredentials(ra *schema.RuntimeApp, root string) (*ProcessCredentials, error) {
+	var c ProcessCredentials
+	var err error
+
+	// mock up a pod so we can call ParseUserGroup
+	pod := &stage1commontypes.Pod{
```

Comment:
Why do you want to mock a pod here instead of using the real one you got in main? This is generally dangerous because of parameters implicitely initialized to their default value.